checksum/host/src/app.c

```diff
@@ -39,14 +39,20 @@
 // changing any file location, just uncomment the following define:
 #define DPU_LOG_DIRECTORY "log/host/"
 
-#define NB_OF_DPUS 8
```

Comment:
All target types do not have the same number of DPUs per rank, thus this program will not have the same output depending on the target. However, the logic to allocate and manage the correct number of DPUs can be a little cumbersome for such a simple example...

Reply:
Is my current logic wrong or too complicated ?
We need to discuss this one.